examples/tv-casting-app/APIs.md

```diff
@@ -152,10 +148,13 @@ client's lifecycle:
     ```
 
 2.  **Commissioning Data** - This object contains the passcode, discriminator,
-    etc which identify the app and are provided to the `CastingPlayer` during
-    the commissioning process. Refer to the Matter specification's
-    [Onboarding Payload](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/qr_code/OnboardingPayload.adoc#ref_OnboardingPayload)
-    section for details on commissioning data.
+    etc. which identify the app and are provided to the `CastingPlayer` during
+    the commissioning process. A Passcode SHALL be included as a 27-bit unsigned
```

Comment:
it is a bit unexpected to see SHALL in all caps since this is a tech doc and not a specification. Are these quotes from the spec?

Reply:
Yes, the statements about passcode, discriminator and rotating device id unique id are from the spec.